package.json

```diff
@@ -1,36 +1,36 @@
 {
```

Comment:
- Remove the whitespace changes.
- I think some libs are missing (typescript, ts-node)?